.travis.yml

```diff
@@ -0,0 +1,27 @@
+language: dart
+sudo: false
+dart:
+  - dev
+  - stable
+  - 1.24.2
+  - 1.23.0
+  - 1.22.1
```

Comment:
I don't think it's worth testing more than dev or stable here. It'll just add to the iteration time without any substantial gain.